components/secret-mgt/org.wso2.carbon.identity.secret.mgt.core/src/main/java/org/wso2/carbon/identity/secret/mgt/core/model/ResolvedSecret.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.com).
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.secret.mgt.core.model;
+
+public class ResolvedSecret extends Secret {
+
+    private String resolvedSecretValue;
+
+    public void setResolvedSecretValue(String resolvedSecretValue) {
+
+        this.resolvedSecretValue = resolvedSecretValue;
+    }
+
+    public String getResolvedSecretValue() {
+
+        return resolvedSecretValue;
+    }
+
```

Comment:
Unnecessary newline

Reply:
Addressed with the commit [4191a12](https://github.com/wso2/carbon-identity-framework/pull/3762/commits/4191a12ce26a59b9a12028b142a6964aeb4ddd2a)